### PR TITLE
Update release notes draft to version v1.21.0-beta.0

### PR DESCRIPTION
**Pull request by pmmalinov01**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.21.0-beta.0

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

File: releases/release-1.21/release-notes/maps/pr-98201-map.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pr: 98201
+releasenote:
+  text: Fixes a duplicate error messages when runing kubectl edit on `ResourceQuota`
```
Comment (wilsonehusin) [inline releases/release-1.21/release-notes/maps/pr-98201-map.yaml:3]:
```suggestion
  text: Fixes a duplicate error messages when running `kubectl edit` on `ResourceQuota`
```

File: releases/release-1.21/release-notes/maps/pr-98458-map.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+pr: 98458
+releasenote:
+  text: Cluster admins can now turn off `/debug/pprof` and `/debug/flags/v` endpoint
+    in kubelet by setting `enableProfilingHandler` and `enableDebugFlagsHandler` to
+    `false` in the Kubelet configuration file. Options `enableProfilingHandler` and
+    `enableDebugFlagsHandler` can be set to `true` only when enableDebuggingHandlers
+    is also set to `true`.
```
Comment (wilsonehusin) [inline releases/release-1.21/release-notes/maps/pr-98458-map.yaml:7]:
```suggestion
  text: Cluster admins can now turn off `/debug/pprof` and `/debug/flags/v` endpoint
    in kubelet by setting `enableProfilingHandler` and `enableDebugFlagsHandler` to
    `false` in the Kubelet configuration file. Options `enableProfilingHandler` and
    `enableDebugFlagsHandler` can be set to `true` only when `enableDebuggingHandlers`
    is also set to `true`.
```

File: releases/release-1.21/release-notes/maps/pr-98678-map.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+pr: 98678
+releasenote:
+  text: The `TTLAfterFinished` is now beta and enabled by default
```
Comment (wilsonehusin) [inline releases/release-1.21/release-notes/maps/pr-98678-map.yaml:3]:
```suggestion
  text: The `TTLAfterFinished` feature flag is now beta and enabled by default
```

File: releases/release-1.21/release-notes/maps/pr-98730-map.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+pr: 98730
+releasenote:
+  text: Several flags related to the deprecated dockershim which are present in the
+    kubelet command line are now showing warning about the future deprecation.
```
Comment (wilsonehusin) [inline releases/release-1.21/release-notes/maps/pr-98730-map.yaml:4]: dockershim deprecation already happened, just not removed 😄 deprecated = will be removed in future release

```suggestion
  text: Kubelet command line flags related to dockershim are now showing deprecation 
    message as they will be removed along with dockershim in future release.
```

File: releases/release-1.21/release-notes/maps/pr-98878-map.yaml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+pr: 98878
+releasenote:
+  text: Turn `CronJobControllerV2` on by default.
```
Comment (wilsonehusin) [inline releases/release-1.21/release-notes/maps/pr-98878-map.yaml:3]:
```suggestion
  text: The `CronJobControllerV2` feature flag graduates to Beta and set to be enabled by default.
```

File: releases/release-1.21/release-notes/maps/pr-98969-map.yaml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+pr: 98969
+releasenote:
+  text: 'Upgrades `IPv6Dualstack` to `Beta` and turns it on by default. New clusters
+    or existing clusters are not be affected until an actor starting adding secondary
+    pods and service cidrs cli flags as described here: [IPv4/IPv6 Dual-stack](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/563-dual-stack)'
```
Comment (wilsonehusin) [inline releases/release-1.21/release-notes/maps/pr-98969-map.yaml:5]:
```suggestion
  text: 'Upgrades `IPv6Dualstack` to `Beta` and turns it on by default. New clusters
    or existing clusters are not be affected until an actor starting adding secondary
    Pods and service CIDRs CLI flags as described here: [IPv4/IPv6 Dual-stack](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/563-dual-stack)'
```

File: releases/release-1.21/release-notes/maps/pr-99068-map.yaml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+pr: 99068
+releasenote:
+  text: |-
+    Unable read directory `/root/bin` from your PATH: open `/root/bin`: no such file or directory. Skipping...
+    error: unable to find any kubectl plugins in your PATH
```
Comment (wilsonehusin) [inline releases/release-1.21/release-notes/maps/pr-99068-map.yaml:5]: image

I think this is a case of no release note is needed :smile:
```suggestion
releasenote:
  do_not_publish: true
```

File: releases/release-1.21/release-notes/maps/pr-99194-map.yaml [hunk 8]
```diff
@@ -0,0 +1,4 @@
+pr: 99194
+releasenote:
+  text: During `kubeamd init` and `kubeamd join` a preflight check does a validation
+    on the host/node name and throw warnings if a name is not compliant
```
Comment (wilsonehusin) [inline releases/release-1.21/release-notes/maps/pr-99194-map.yaml:4]:
```suggestion
  text: 'kubeadm: perform pre-flight validation on host/node name upon
    `kubeadm init` and `kubeadm join`, showing warnings on non-compliant names'
```

**Review by wilsonehusin — COMMENTED** [review]
awesome work! :tada: some changes while we're at it...

**Comment by wilsonehusin** [comment]
/uncc divya-mohan0209

/cc sladyn98
(just FYI since you ran the first review for `beta.0` 😄)

**Comment by wilsonehusin** [comment]
blerp, cc @sladyn98 (just FYI since you ran the first review for `beta.0` :smile:)

File: releases/release-1.21/release-notes/maps/pr-98021-map.yaml [hunk 9]
```diff
@@ -0,0 +1,4 @@
+pr: 98021
+releasenote:
+  kinds:
+  - deprecation
```
[the remark this answers is not in the document]
Comment (pmmalinov01) [reply releases/release-1.21/release-notes/maps/pr-98021-map.yaml:4]: Should I remove the file?

File: releases/release-1.21/release-notes/maps/pr-98730-map.yaml (hunk 4 above)
Comment (pmmalinov01) [reply releases/release-1.21/release-notes/maps/pr-98730-map.yaml:4]: I had wonder how to phrase this one a lot. Thanks, it sounds better than.

File: releases/release-1.21/release-notes/maps/pr-98033-map.yaml [hunk 10]
```diff
@@ -0,0 +1,4 @@
+pr: 98033
+releasenote:
+  text: The feature gate `RootCAConfigMap` graduated to GA in `v1.21` and will be
+    removed in `v1.22`
```
[the remark this answers is not in the document]
Comment (sladyn98) [reply releases/release-1.21/release-notes/maps/pr-98033-map.yaml]: Yeah the versions can be kept normal, as far as I have seen the style guide

File: releases/release-1.21/release-notes/maps/pr-99068-map.yaml (hunk 7 above)
Comment (pmmalinov01) [reply releases/release-1.21/release-notes/maps/pr-99068-map.yaml:5]: Yes, it is a trivial change. Will remove it

File: releases/release-1.21/release-notes/maps/pr-98021-map.yaml (hunk 9 above)
[the remark this answers is not in the document]
Comment (sladyn98) [reply releases/release-1.21/release-notes/maps/pr-98021-map.yaml:4]: I guess you can remove this file

File: releases/release-1.21/release-notes/maps/pr-98730-map.yaml [hunk 11]
```diff
@@ -0,0 +1,5 @@
+pr: 98730
+releasenote:
+  text: Kubelet command line flags related to dockershim are now showing deprecation 
+    message as they will be removed along with dockershim in future release.
+
```
Comment (sladyn98) [inline releases/release-1.21/release-notes/maps/pr-98730-map.yaml:5]: Is this a deliberate newline ?

File: releases/release-1.21/release-notes/maps/pr-98969-map.yaml [hunk 12]
```diff
@@ -0,0 +1,5 @@
+pr: 98969
+releasenote:
+  text: 'Upgrades `IPv6Dualstack` to `Beta` and turns it on by default. New clusters
+    or existing clusters are not be affected until an actor starting adding secondary
```
Comment (sladyn98) [inline releases/release-1.21/release-notes/maps/pr-98969-map.yaml:4]:
```suggestion
    or existing clusters are not be affected until an actor starts adding secondary
```

Comment (pmmalinov01) [reply releases/release-1.21/release-notes/maps/pr-98969-map.yaml:4]: Good catch! Thanks

File: releases/release-1.21/release-notes/maps/pr-98730-map.yaml (hunk 11 above)
Comment (pmmalinov01) [reply releases/release-1.21/release-notes/maps/pr-98730-map.yaml:5]: Addressed, thanks.

**Comment by sladyn98** [comment]
Great work @divyenpatel :tada: 
/lgtm

File: releases/release-1.21/release-notes/maps/pr-98021-map.yaml (hunk 9 above)
[the remark this answers is not in the document]
Comment (pmmalinov01) [reply releases/release-1.21/release-notes/maps/pr-98021-map.yaml]: let us see I wanted to remove it too.

**Comment by wilsonehusin** [comment]
wonderful!

/lgtm
/approve

**Pull request merged**